### PR TITLE
Dropdown changes

### PR DESCRIPTION
**Pull request by ghislaineguerin**
@alex-deaconu please start working with these styles for the dropdown changes. the html is on the index.html file. I will add comments.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,6 +77,46 @@
     <img src="https://lh3.googleusercontent.com/-q9MKVxrwf6Y/VFulraKuFNI/AAAAAAAAAC4/r0Xl2q5YDmI/w280-h280-p/10734133_10154773862775246_6527835082350492155_n.jpg" class="profile-pic" width="30" height="30" alt="User" />
   </a>
 
+  <ul class="dropdown-menu">
+
+    <li class="dropdown-header sub-company-header">
```
Comment (ghislaineguerin) [inline index.html]: @alex-deaconu this class .sub-company-header must be included to make the background red.

File: index.html [hunk 2]
```diff
@@ -77,6 +77,46 @@
...
+      <p>You are in Sub-Company <strong>Abc</strong>
+      <span class="text-muted">|</span>
+       <a href="#">Change</a></p>
+      <p><a href="#">Switch to My Company</a></p>
+    </li>
+
+    <li class="dropdown-header">
```
Comment (ghislaineguerin) [inline index.html]: This is the otherwise normal header.

File: index.html [hunk 3]
```diff
@@ -77,6 +77,46 @@
...
+      </a>
+    </li>
+    <li>
+      <a href="" class="action" ng-click="paymentMethods()">
+        <i class="fa fa-usd icon-left"></i>
+        <span class="item-name">Payment Methods</span>
+      </a>
+    </li>
+
+    <li class="dropdown-footer">
```
Comment (ghislaineguerin) [inline index.html]: @alex-deaconu new footer class for the dropdown action button.

File: index.html [hunk 4]
```diff
@@ -122,5 +162,54 @@
   <p class="text-center add-top"><strong>Wrong User Account or Company?</strong> <a href="#">Sign Out</a> and Sign back in with the correct User or <a href="#">Select a Sub-Company</a>.</p>
 </div><!--container-->
 
+
+<div class="action-sheet user-profile-dropdown ng-scope is-action-sheet-opened"><!-- ngInclude: undefined --><ng-include src="templateUrl" class="ng-scope"><!-- ngIf: isRiseVisionUser --><!-- ngInclude: undefined --><li class="dropdown-header ng-scope" ng-show="!isSubcompanySelected">
```
Comment (ghislaineguerin) [inline index.html]: @alex-deaconu needs the user-profile-dropdown class

File: index.html [hunk 5]
```diff
@@ -122,5 +162,54 @@
...
+</li>
+<li class="divider ng-scope ng-hide" ng-show="false"></li>
+<li ng-show="false" class="ng-scope ng-hide">
+  <a href="" class="action" ng-click="paymentMethods()">
+    <i class="fa fa-usd"></i>
+    <span class="item-name">Payment Methods</span>
+  </a>
+</li>
+
+<li class="dropdown-footer text-right ng-scope" ng-show="isLoggedIn">
```
Comment (ghislaineguerin) [inline index.html]: @alex-deaconu needs text-right class to position the button

**Comment by alex-deaconu** [comment]
@ghislaineguerin squashed and ready to merge. Thanks.

**Pull request merged**